cmake/Macros.cmake

```diff
@@ -35,6 +35,17 @@ function(sfml_set_stdlib target)
             message(FATAL_ERROR "Clang is the only supported compiler on macOS")
         endif()
     endif()
+
+    # on some platforms (e.g. ARM), GCC requires linking libatomic to use <atomic> features
+    if(SFML_COMPILER_GCC)
+        include(CheckCXXSourceCompiles)
+        check_cxx_source_compiles("#include <atomic>
+            #include <cstdint>
+            int main(){std::atomic<std::uint64_t> x(1); return x.fetch_add(1);}" ATOMIC_TEST)
```

Comment:
Minor knit but if you replaced `std::uint64_t` with `int` you could drop `<cstdint>`. Presumably it's `fetch_add` that causes issues, not that we used a 64 bit unsigned int.

Reply:
Depending on whether the operation is translated into a compiler intrinsic or a function implementation, linking to the external library might not be required. Since this is what we use in the code, I figured it is safer to test for this exact code even though your version would be shorter. I can build in the shorter version and see what happens.